app/styles/libs/_helpers.scss

```diff
@@ -98,3 +98,15 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.search-bar {
```

Comment:
This class is already defined

Reply:
Either use diff class name

Reply:
it is breaking other styling. Pls revert

Reply:
changed class name.